### PR TITLE
feat(Xero.node): add tracking categories

### PR DESCRIPTION
**Pull request by abourge**

### CODE REVIEW

File: packages/nodes-base/nodes/Xero/Xero.node.ts [hunk 1]
```diff
@@ -296,6 +296,7 @@ export class Xero implements INodeType {
 							lineItem.TaxAmount = lineItemValue.taxAmount as string;
 							lineItem.TaxType = lineItemValue.taxType as string;
 							lineItem.UnitAmount = lineItemValue.unitAmount as string;
+							lineItem.Tracking = lineItemValue.tracking as [];
```
Comment (molenchuk) [inline packages/nodes-base/nodes/Xero/Xero.node.ts]: @abourge what about line 288? I think there is a Tracking object there, could we set this there or should we not do that?

**Review by molenchuk — CHANGES_REQUESTED** [review]

File: packages/nodes-base/nodes/Xero/Xero.node.ts (hunk 1 above)
Comment (abourge) [reply packages/nodes-base/nodes/Xero/Xero.node.ts]: @molenchuk Good point, changed it

File: packages/nodes-base/nodes/Xero/Xero.node.ts [hunk 2]
```diff
@@ -285,7 +285,7 @@ export class Xero implements INodeType {
 						const lineItems: ILineItem[] = [];
 						for (const lineItemValue of lineItemsArray) {
 							const lineItem: ILineItem = {
-								Tracking: [],
+								Tracking: lineItemValue.tracking as [],
```
Comment (Ken-Michalak) [inline packages/nodes-base/nodes/Xero/Xero.node.ts]: That doesn't seem like the right type. What is `tracking`? I'm thinking you either want `as string[]` or `[lineItemValue.tracking]`. `[]` is an empty tuple that doesn't need a type.

**Review by Ken-Michalak — CHANGES_REQUESTED** [review]

File: packages/nodes-base/nodes/Xero/Xero.node.ts (hunk 2 above)
Comment (molenchuk) [reply packages/nodes-base/nodes/Xero/Xero.node.ts]: @Ken-Michalak according to the Xero node's Interface file Tracking is marked as a `Tracking?: IDataObject[];`

Comment (abourge) [reply packages/nodes-base/nodes/Xero/Xero.node.ts]: @Ken-Michalak updated

**Review by Ken-Michalak — APPROVED** [review]

**Pull request merged**